data_model/src/query/mod.rs

```diff
@@ -806,28 +807,28 @@ pub mod asset {
 
     impl FindAssetById {
         /// Construct [`FindAssetById`].
-        pub fn new(id: impl Into<EvaluatesTo<AssetId>>) -> Self {
+        pub fn new(id: impl Into<AssetId>) -> Self {
```

Comment:
IMO all these constructors can be derived now. I don't see much point in having `Into<AssetId>` or similar